string request
---
spectrometer_drivers/SpectraArray response